operators/src/engine/clonable_operator.rs

```diff
@@ -74,6 +74,7 @@ pub trait CloneableInitializedVectorOperator {
 }
 
 /// Helper trait for making boxed `InitializedPlotOperator`s cloneable
+#[allow(dead_code)]
```

Comment:
maybe just pub use it like the other CLoneable... traits

Reply:
[6849724](https://github.com/geo-engine/geoengine/pull/973/commits/684972467e851106b49dca727adc36c1d5595550)